generators/gulp/templates/gulpfile.js

```diff
@@ -42,7 +42,8 @@ gulp.task('test', ['pre-test'], function (cb) {
 
   gulp.src('test/**/*.js')
     .pipe(plumber())
-    .pipe(mocha({reporter: 'spec'}))
+    .pipe(mocha({reporter: 'spec'<% if (babel) { %>,
+      require: ['babel/register']<% } %>}))
```

Comment:
@stevemao @amitport Using that snippet we end up with a trailing comma (and notice that it's the wrong indentation)